lib/sidekiq/form526_backup_submission_process/submit.rb

```diff
@@ -52,6 +52,10 @@ class Submit
           'Form 526 Backup Submission Retries exhausted',
           { job_id:, error_class:, error_message:, timestamp:, form526_submission_id: }
         )
+
+        if Flipper.enabled?(:send_backup_submission_exhaustion_email_notice)
+          ::Form526SubmissionFailureEmailJob.perform_async(form526_submission_id: form526_submission_id)
+        end
       rescue => e
         ::Rails.logger.error(
           'Failure in Form526BackupSubmission#sidekiq_retries_exhausted',
```

Comment:
Do you need to add a ` StatsD.increment('silent_failure', tags: DD_ZSF_TAGS)` in this rescue? (IE if there is a failure to queue the email job, there is a silent failure)

Reply:
I thought about that... I guess my thougt was that the exhaustion hook is a failure condition as is. So technically, that's true of everything in here. It's the elevator brakes problem... if our failsafe blows up, we don't want to loose that last ditch logging because the email job blew up, right? 

I think we should leave this rescue block alone. If anything fails to send in the exhaustion hook, it will be caught in the saftey net

Reply:
IMHO this is for rescuing the rescue. We shouldn't move business logic into it

Reply:
i guess if it's just the stats d increment it's nbd tho 🤔

Reply:
added